src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/ImportControlCheckTest.java

```diff
@@ -275,6 +279,57 @@ public void testUrlIncorrectUrl() throws Exception {
     }
 
     @Test
+    public void testResource() throws Exception {
+        final DefaultConfiguration checkConfig = createCheckConfig(ImportControlCheck.class);
+        checkConfig.addAttribute("file",
+                "/com/puppycrawl/tools/checkstyle/checks/imports/import-control_one.xml");
```

Comment:
Why we declare full path when you created the method `getResourcePath` to make this simple?
Lets change this to use the new method.